src/FSM/tests/Test/Runner/BacktrackingRunnerTest.php

```diff
@@ -15,7 +15,7 @@ class BacktrackingRunnerTest extends TestCase
      */
     private $fsm;
 
-    protected function setUp()
```

Comment:
PHP 8 does not allow incompatible method overrides. As the parent defined the `void` return type, it has to be added here too.